### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by partha8**
## Note
604 additions include 274 lines of CSS which has already been reviewed and approved. Kindly ignore them and please review the cart functionalities.

## files to check
- [src/hooks/useGetCart.js](https://github.com/partha8/algern-clothing-react/compare/cart?expand=1#diff-7b0c9aced9e47453b28c4d7b81aeccae10f9dd0cc44f6022340db3dd1fcc5f31) -  custom hook to fetch cart array everytime a user logs in
- [src/pages/Cart/Cart.js](https://github.com/partha8/algern-clothing-react/compare/cart?expand=1#diff-ef9e5d30776411080a44497695290f3566d65ab41c39b05cb39e8d072248021b)
- [src/reducers/productsReducer.js](https://github.com/partha8/algern-clothing-react/compare/cart?expand=1#diff-1e72764740711c06e4f3cf410674a7d5a3bb67e5236ee138c5019d857f828e3a)
- [src/utils/productUtils.js](https://github.com/partha8/algern-clothing-react/compare/cart?expand=1#diff-316a67f1af7813b6dd0bddc5262ea859f542723384a44aeb1e8daed288ab6aa7)

## files to avoid
- [ ] cart.css

## Future changes
Breakdown the cart page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ export const App = () => {
   useGetProducts();
   useGetCategories();
   useGetWishlist();
+  useGetCart();
   // useSignup();
```
Comment (b0llu) [inline src/App.js:29]: remove this commented function

**Review by b0llu — APPROVED** [review]
Just suggested one thing everything else looks good

File: src/utils/productUtils.js [hunk 2]
```diff
@@ -60,4 +59,107 @@ export const addToWishlist = async (
...
+      }
+    );
+    if (response.status === 200) {
+      productsDispatch({
+        type: "SET_CART",
+        payload: response.data.cart,
+      });
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (ApurvaSawant11) [inline src/utils/productUtils.js:133]: You can use `console.error()`

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me!

**Pull request merged**